install.php

```diff
@@ -181,8 +181,6 @@ function ArraySearchRecursive($Needle,$Haystack,$NeedleKey="",$Strict=false,$Pat
 		exit;
 		$rightserror=1;
 	}else{ // so we have users and at least one site admin
-		require_once("customers.inc.php");
```

Comment:
Pretty sure this needs to be a copy of the line above pulling in the the people and user class.